### PR TITLE
Maintaining a Liferay DXP Installation: Backing Up and Patching Liferay DXP

### PR DESCRIPTION
**Pull request by jhinkey**
Maintaining a Liferay DXP Installation:
- Backing Up
- Patching Liferay DXP (all the patching articles)

### CODE REVIEW

File: docs/dxp/7.2.x/en/installation-and-upgrades/05-maintaining-a-dxp-installation/09-working-with-patches.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+If you removed certain patches or there is a collision of some kind use the"-force" argument to force the Patching Tool to install the currently available patches no matter what.
+
+Example:
+
+```bash
+patching-tool profile_name install -force
+```
+
+## Uninstalling Patches
```
Comment (jhinkey) [inline docs/dxp/7.2.x/en/installation-and-upgrades/05-maintaining-a-dxp-installation/09-working-with-patches.md]: @CsabaTurcsan - Would you please review this section on Uninstalling Patches?

File: docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/01-installing-liferay-on-an-application-server/01-installing-liferay-on-tomcat.md [hunk 2]
```diff
@@ -93,8 +93,11 @@ DXP depends on many JARs included by DXP Tomcat bundle. Some of the bundle's JAR
   * `configs`
   * `core`
   * `marketplace`
-  * `target-platform`
-  * `test`
+  * `modules`
+  * `portal`
+  * `state`
+  * `static`
+  * `war`
```
Comment (jhinkey) [inline docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/01-installing-liferay-on-an-application-server/01-installing-liferay-on-tomcat.md]: @justinchoi001 FYI - The `osgi/` subfolders were updated in 7.2. https://issues.liferay.com/browse/LRDOCS-7509

Comment (justinchoi001) [reply docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/01-installing-liferay-on-an-application-server/01-installing-liferay-on-tomcat.md]: Thanks Jim! I will update the Wildfly and WebLogic PRs accordingly #76  and #77

**Comment by jhinkey** [comment]
I'll send another pull after resolving the conflicts.

**Comment by jhinkey** [comment]
I'm going to merge/meld with Support's docs on this.

**Pull request closed without merging**